dts/riscv/rapidsilicon/rapidsi_virgo.dtsi

```diff
@@ -181,12 +181,18 @@
 			 status = "disabled";
 		 };
  
-		 pufcc: pufcc@A0130000 {
+		 pufs: pufcc@A0130000 {
```

Comment:
There are some unnecessary spaces that dont look pretty. They are in addition to the tabs. I would appreciate it if they are removed

Reply:
I will remove them in the xCB pull request in both files (dtsi and dts).